### ISSUE TITLE
big sur keyboard

### ISSUE THREAD

**Issue by electriquo** [Clipy/Clipy#428]
firstly, i wanna thank you for the clipy.

i am using clipy version 1.2.1 and i've upgraded to big sur.
now (it used to work prior to the big sur upgrade) when i access the clipy history using a keyboard shortcut, i am unable to select from history using the keyboard (arrows and number keys).

**Comment by curranjw** [comment]
In addition to this issue, input "COMMAND" + "SHIFT" stopped working when you click the item in the history or snippet.

**Comment by electriquo** [comment]
i was able to bypass the issue by restarting clipy (quitting clipy and starting it again).

does it work for you?

**Comment by electriquo** [comment]
i also noticed that even when i bypass the issue by restarting clipy, the issue reoccur when the mouse is hovering the history list

**Comment by drej1** [comment]
Same occured to me on Big Sur with M1 SoC. Restart helped but I think it will occur again...

**Comment by ohassidi** [comment]
> i was able to bypass the issue by restarting clipy (quitting clipy and starting it again).
> 
> does it work for you?

Still annoying to restart the app, each time

**Comment by AmiciA** [comment]
I am still having this issue with Monterey 12.0.1 and Clipy v1.2.1. Keyboard navigation with arrow keys works only after a restart of Clipy.

**Comment by electriquo** [comment]
to all those who follow this thread, look at https://github.com/Clipy/Clipy/issues/319#issuecomment-874115468

**Comment by Econa77** [comment]
This issue may have been fixed by macOS updates since it was originally reported.

If you are still using Clipy and the same issue still occurs with the latest version of Clipy, please create a new issue with the details.

Sorry that we were not able to respond or work on this for such a long time. Development of Clipy has now resumed. Please see #590 for more details.
